editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -6739,6 +6780,10 @@ Node3DEditor::Node3DEditor(EditorNode *p_editor) {
 	tool_button[TOOL_MODE_SCALE]->set_shortcut(ED_SHORTCUT("spatial_editor/tool_scale", TTR("Scale Mode"), KEY_R));
 	tool_button[TOOL_MODE_SCALE]->set_shortcut_context(this);
 
+	tool_button[TOOL_MODE_EXTERNAL] = memnew(Button);
```

Comment:
This button isn't in scene tree, so it must be added to it with `add_child` or just deleted in destructor.

Reply:
Ah! I was looking for what was causing the leak. I don't want to add that button to the scene, so I made it free in the destructor. Thank you very much!